Recipes/CGC/apps_copyFromMyProject.ipynb

```diff
@@ -90,7 +90,7 @@
     "# [USER INPUT] Set project name; project (p_) and file (f_) indices here:\n",
     "source_project_name = 'Keep on Smiling'\n",
     "my_project_name = 'Life is Beautiful'\n",
-    "app_id = 'jack_digi/keep-on-smiling/TorrentMappingAlignmentProgramMap4/0'\n",
```

Comment:
No, I will change it to one with revision number.
The revision number within app ID causes an error as the app ID is not recognizable anymore. Revision number has to be given as a separate argument to the method now. I will change it.